src/main/java/flow/Sigel.java

```diff
@@ -22,65 +21,92 @@
 
 	/** @param args Not used */
 	public static void main(String... args) {
-		morphSigelDump();
-		morphSigelUpdates("2013-01-01", "2013-12-31", "sigel-updates2013.out.json");
-		morphSigelUpdates("2014-01-01", getToday(), "sigel-updates2014.out.json");
-	}
+		final FileOpener dumpOpener = setupSigelMorphDump();
+		final OaiPmhOpener update1Opener =
+				setupSigelMorphUpdate("2013-01-01", "2013-12-31",
+						"sigel-updates2013.out.json");
+		final OaiPmhOpener update2Opener =
+				setupSigelMorphUpdate("2014-01-01", "2014-12-31",
+						"sigel-updates2014.out.json");
+		final OaiPmhOpener update3Opener =
+				setupSigelMorphUpdate("2015-01-01", Helpers.getToday(),
+						"sigel-updates2015.out.json");
 
```

Comment:
Hm, starting with 2016 this program will ignore new sigel data.
It should be configurable to steer the program from the outside, to be usable from schedulers like cron.
Also, is it really necessary to get all the data all over again ? What if we want daily updates (this is what the actual lobid-organisations does) - it would start to download all the data day by day.
IMO the data should be stored ("cached") in filesystem or db. Makes you also more independent. And is easier for debugging purposes.